smarts/core/trap_manager.py

```diff
@@ -286,14 +312,21 @@ def largest_vehicle_plane_dimension(vehicle: Vehicle):
                     trap.default_entry_speed,
                     social=agent_id in sim.agent_manager.pending_social_agent_ids,
                 )
+            elif trap_condition and capture.vehicle_id is not None:
```

Comment:
Oh, I just noticed that pytype did not catch that this was a bug since `trap_condition` could be unbound.